spec/controllers/work_packages/creation_spec.rb

```diff
@@ -0,0 +1,120 @@
+#-- encoding: UTF-8
+#-- copyright
+# OpenProject is a project management system.
+# Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See doc/COPYRIGHT.rdoc for more details.
+#++
+
+require 'spec_helper'
+require 'support/shared/previews'
+
+describe WorkPackagesController, type: :controller do
+  let(:user)         { FactoryGirl.create :user, admin: true }
+  let(:project)      { FactoryGirl.create :project, is_public: true }
+
+  before do
+    allow(User).to receive(:current).and_return user
+    expect(controller).to receive(:authorize).and_return(true)
+  end
+
+  around(:each) do |example|
+    delay = Delayed::Worker.delay_jobs
```

Comment:
It's definitely false in test mode. Just figured it would be more flexible this way. Granted, it's highly unlikely that the value will ever change to true. So I suppose I could just use false directly.